New package: OptimizationCallbacks v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: OptimizationCallbacks
- Repository: https://github.com/jbrea/OptimizationCallbacks.jl
- Created by: @jbrea
- Version: v0.1.0
- Commit: 55682cccea02cc407fc37c95bce079fca91fb389
- Reviewed by: @jbrea
- Reference: https://github.com/jbrea/OptimizationCallbacks.jl/commit/55682cccea02cc407fc37c95bce079fca91fb389#commitcomment-149697539
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd68ff4887c31d9bfb248bdab0a1ebc9aa46b127e76d7fe94d5aea2884f5b2bbedd75e793d2f85daadbf815c0a2f12bdc038cc84a17071ac3aef7429c34849adb03063581b82c5e3340031e159ff8a64b27ac8573b8f9d19a003dca6f13b7d2f931beea230262b8d0f6669f83d000aa6f77b1e31d9552ae6867a90fa53efded48ec56bac2db279bd26f782609c716975bd12e88c31d545206fbbcad91d46ae4d337d3dc43fb05230c8b25379ec0e8fc9159 -->

goerz commented:
Congratulations on the new package! Could you maybe a little bit of context to the README and/or the documentation? When would someone use this package? Is it intended to work with one of Julia's existing optimization frameworks?

JuliaRegistrator committed:
New package: OptimizationCallbacks v0.1.0

UUID: 65fa8838-c146-46ed-aad7-5f6b0eb2af66
Repo: https://github.com/jbrea/OptimizationCallbacks.jl.git
Tree: 77f6d67ac81c7be8ce8c32bc357417c38e20a9ac

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

jbrea commented:
Thanks for the question @goerz . I added some explanations to the README/docs. Does this help?

goerz commented:
[noblock] Looks good!

Make sure to add `[noblock]` to your comment
